lib/vaahextendflutter/services/storage/storage.dart

```diff
@@ -0,0 +1,132 @@
+import '../../env.dart';
+import 'local/flutter_secure_storage/flutter_secure_storage_impl.dart';
+import 'local/hive/hive_storage_impl.dart';
+
+abstract class Storage {
+  static final EnvironmentConfig _envConfig = EnvironmentConfig.getEnvConfig();
+
+  ///Creates a new Local Storage [HiveStorageImpl] or [FlutterSecureStorageImpl]
+  ///
+  ///The argument [name] is used to open Hive box with that name.
+  ///
+  ///If you don't provide [name], 'default' will be used.
+  factory Storage.createLocal({String name = 'default'}) {
+    switch (_envConfig.localStorageType) {
+      case LocalStorageType.hive:
+        final hive = HiveStorageImpl(name: name);
+        hive.init();
+        return hive;
+      case LocalStorageType.flutterSecureStorage:
+        return FlutterSecureStorageImpl();
```

Comment:
call init on this too, all classes which implements base class should have init (even if it is empty methos)

Reply:
Refer https://github.com/webreinvent/vaahflutter/pull/43#discussion_r1631920185